platforms/joystick/joystick_adaptor_test.go

```diff
@@ -31,8 +30,9 @@ func TestAdaptorConnect(t *testing.T) {
 	a := initTestAdaptor()
 	gobottest.Assert(t, a.Connect(), nil)
```

Comment:
```suggestion
	assert.Nil(t, a.Connect())
```
my suggestion is to drop usage of gobottest in this small file and use "github.com/stretchr/testify/assert" instead, see #960

Reply:
Done.